src/main/java/com/ninjaguild/dragoneggdrop/commands/DragonTemplateCmd.java

```diff
@@ -17,6 +12,10 @@
 import org.bukkit.entity.Player;
 import org.bukkit.inventory.ItemStack;
 
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
```

Comment:
Unnecessary changes to import order, though the change to NumberUtils (lang3 vs lang) is fine.

Reply:
That is the standard way IntelliJ sorts it. I don't really see a reason to change this especially as the previous version also doesn't have an apparent pattern for me.